src/main/java/lotto/domain/Winners.java

```diff
@@ -24,26 +16,11 @@ public Statistics match(Orders orders) {
         Statistics statistics = new Statistics();
 
         orders.getOrders().stream()
-                .map(this::matchWinningRule)
-                .filter(Objects::nonNull)
+                .map(orderNumbers -> WinningRule.match(this.winNumbers, orderNumbers, this.bonusNumber))
+                .filter(n -> n != WinningRule.NONE)
                 .forEach(n -> statistics.put(n, statistics.get(n) + 1));
```

Comment:
당첨 결과는 항상 누적만 하니 증가 시키는 로직은 Statistics에서 수행하면 어떨까요?
```suggestion
                .forEach(n -> statistics.put(n));
```